File: C#/LocalSearchOptimization/Examples/Structures/TwoOperands.cs
using LocalSearchOptimization.Components;

namespace LocalSearchOptimization.Examples.Structures

{
    public class TwoOperands : Configuration
    {
        public int First { get; }

        public int Second { get; }

        public TwoOperands(int i, int j, Operator operation) : base(operation)
        {
            First = i;
            Second = j;
        }
    }
}

File: java/src/localsearchoptimization/examples/structures/TwoOperands.java
package localsearchoptimization.examples.structures;

import localsearchoptimization.components.Configuration;
import localsearchoptimization.components.Operator;

public class TwoOperands extends Configuration {

    public int first;

    public int second;

    public TwoOperands(int i, int j, Operator operation) {
        super(operation);
        first = i;
        second = j;
    }
}